mitodl/salt-ops
# add GTM keys for MITxOnline RC

#### What are the relevant tickets?
Related to https://github.com/mitodl/mitxonline/issues/73

#### What's this PR do?
- Adds values GTM related keys
    - `GOOGLE_TRACKING_ID `
    - `GOOGLE_TAG_MANAGER_ID `

#### How should this be manually tested?
Once they are part of RC release of MITxOnline, we should start seeing the page events. Or at least we should see a network request to google analytics e.g. something like `https://www.google-analytics.com/j/collect?`
